README.dist.md

```diff
@@ -34,21 +29,21 @@ If you update Vagrant, you may need to update your vagrant plugins with `vagrant
 
   ```
     composer install
-    vagrant up
+    ddev start
   ```
-3. You will be prompted for the administration password on your host machine
-4. Log in to the virtual machine (the VM): `vagrant ssh`
-5. From within the VM, build and install the Drupal site: `phing install migrate`
-1. Visit your site at [your-project.local](http://your-project.local)
+3. Log in to the ddev environment: `ddev ssh`
+4. From within ddev, build and install the Drupal site: `vendor/bin/phing install migrate`
+5. Visit your site at [your-project.ddev.site](http://your-project.ddev.site)
 
 ## How do I work on this?
 
-You can edit code, update documentation, and run git commands by opening files directly from your host machine.
+You can edit code, update documentation, and run git commands by opening files directly from your machine.
 
-To run project-related commands other than `vagrant up` and `vagrant ssh`:
+To run site management commands like `drush status` or `phing install`:
 
-* SSH into the VM with `vagrant ssh`
-* You'll be in your project root, at the path `/var/www/your-project.local/`
+* Start the development environment with `ddev start`
+* SSH into the development environment with `ddev ssh`
+* You'll be in your project root, at the path `/var/www/html/`
```

Comment:
I found that the default directory is `/var/www/html/web` when I rand `ddev ssh`.